src/Life/Github.hs

```diff
@@ -44,23 +45,23 @@ askToPushka commitMsg = do
     "git" ["diff", "--name-status", "HEAD"]
     continue <- chooseYesNo "Would you like to proceed?"
     if continue
-    then pushka commitMsg
+    then pushka (Branch "master") commitMsg
     else errorMessage "Abort pushing" >> exitFailure
 
 -- | Make a commit and push it.
-pushka :: Text -> IO ()
-pushka commitMsg = do
+pushka :: Branch -> Text -> IO ()
+pushka (Branch br) commitMsg = do
```

Comment:
I think it doesn't make much sense to shorten `branch` to `br`. It's okay to me more verbose.